assets/training/finetune_acft_hf_nlp/components/pipeline_components/nlp_multiclass/spec.yaml

```diff
@@ -260,17 +240,14 @@ jobs:
       huggingface_id: ${{parent.inputs.model_name}}
   preprocess:
     type: command
-    component: azureml:text_classification_datapreprocess:0.0.12
+    component: azureml:nlp_multiclass_datapreprocessing:0.0.1
```

Comment:
can you change the format to refer the components from azureml registry. Like https://github.com/Azure/azureml-assets/blob/909b2fc3cea4beb47036e65ce781085272d1663b/assets/training/finetune_acft_hf_nlp/components/pipeline_components/text_classification/spec.yaml#L474C2-L474C2

Reply:
This is preferred format as it would pull in the components from appropriate registry such as preview, staging or prod.